docs/production/upgrade-or-modify.md

```diff
@@ -239,12 +239,15 @@ instructions for other supported platforms.
 
 5. Finally, we need to reinstall the current version of Zulip, which
    among other things will recompile Zulip's Python module
-   dependencies for your new version of Python:
+   dependencies for your new version of Python, and run a command
+   that will fix any Full-Text Search indexes that may have been
+   corrupted by the PostgreSQL upgrade:
 
     ```
     rm -rf /srv/zulip-venv-cache/*
     /home/zulip/deployments/current/scripts/lib/upgrade-zulip-stage-2 \
         /home/zulip/deployments/current/ --ignore-static-assets
+    su zulip -c '/home/zulip/deployments/current/manage.py audit_fts_indexes'
```

Comment:
Do we want to add a `--audit-search-indexes` option to `upgrade-zulip-stage-2` that calls the management command for us?  That would mean we can avoid adding extra complexity to the documentation.

Reply:
Yeah make sense, fixed!